src/main/java/sonardenemeleri/sonar/App.java

```diff
@@ -7,8 +7,19 @@
 public class App 
 {
     public static void main( String[] args )
-    {
+    {final int deneme;
+	int target2 = -5;
+	int num3 = 3;
+final int deneme2;
+	target2 =- num3;  // Noncompliant; target = -3. Is that really what's meant?
+	target2 =+ num3; // Noncompliant; target = 3
     	
+double d = 1.1;
+
+BigDecimal bd1 = new BigDecimal(d); // Noncompliant; see comment above
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "bd1". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "bd1" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1481)